pkg/asset/installconfig/platform.go

```diff
@@ -114,16 +147,26 @@ func (a *Platform) awsPlatform() (*asset.State, error) {
 
 	return assetStateForStringContents(
 		AWSPlatformType,
-		strings.Split(string(region.Contents[0].Data), " ")[0],
+		strings.SplitN(string(region.Contents[0].Data), " ", 2)[0],
 	), nil
 }
 
 func (a *Platform) libvirtPlatform() (*asset.State, error) {
 	prompt := asset.UserProvided{
-		Prompt: &survey.Input{
-			Message: "URI",
-			Help:    "The libvirt connection URI to be used. This must be accessible from the running cluster.",
-			Default: "qemu+tcp://192.168.122.1/system",
+		Question: &survey.Question{
+			Prompt: &survey.Input{
+				Message: "URI",
+				Help:    "The libvirt connection URI to be used. This must be accessible from the running cluster.",
+				Default: "qemu+tcp://192.168.122.1/system",
+			},
+			Validate: survey.ComposeValidators(survey.Required, func(ans interface{}) error {
+				value := ans.(string)
+				uri, err := url.Parse(value)
+				if uri.Scheme == "" {
```

Comment:
Should we return `err` if non-nil before this check?

Reply:
> Should we return err if non-nil before this check?

Fixed with 362eda8 -> 5b3c205.